alacritty.yml

```diff
@@ -325,9 +325,9 @@
   # Cursor style
   #
   # Values for `style`:
-  #   - ▇ Block
-  #   - _ Underline
-  #   - | Beam
+  #   - ▇ Block or BlinkingBlock
+  #   - _ Underline or BlinkingUnderline
+  #   - | Beam or BlinkingBeam
```

Comment:
I'd prefer to have a separate option for this, like `blink: true`. Here, we mix two different things in a single configuration option.

Maybe in the future, support for custom cursors may be added and someone is crazy enough to want a blinking emoji. That would be easier if I could set something like: `style: '★'` and set the blink option elsewhere.

Reply:
That would work too yes, I went for this kind of settings because of the escape sequences (not handled yet in the current state of that PR) that instruct the terminal to switch to "Block" or "BlinkingBlock", but that's not a blocker.

Reply:
As mentioned previously by @kchibisov, you can actually combine this with the blink rate option.

Reply:
> Maybe in the future, support for custom cursors may be added and someone is crazy enough to want a blinking emoji. That would be easier if I could set something like: style: '★' and set the blink option elsewhere.

That's certainly not desired, but generally you are right. This should just be a separate boolean flag.

Reply:
I can make the `CursorStyle` enum a two-fields struct: `shape` (current enum) and `blinking` (bool).
Or keep the current enum with `Blinking*` variants until alacritty supports more shapes, what would you think is more reasonable @chrisduerr?

Reply:
You should probably have a `CursorShape` enum and a `CursorStyle` struct that has a boolean option if the cursor should be blinking or not.